packages/bpk-component-calendar/src/BpkCalendarGrid-test.js

```diff
@@ -32,7 +32,7 @@ describe('BpkCalendarGrid', () => {
     const tree = renderer
       .create(
         <BpkCalendarGrid
-          month={new Date('2016-10')}
+          month={new Date(Date.UTC(2016, 9, 30, 3, 0, 0))}
```

Comment:
We use a similar method elsewhere (see [composeCalendar-test.js](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-calendar/src/composeCalendar-test.js#L44))wich doesn;t specify hours, mins, seconds or milliseconds ... How come you specify them here?

Reply:
Good point. I'm not sure why, but if I don't specify them one of the tests fails in the Austrailian TZ with the following snapshot difference:
`"focusedDate":"2010-03-01T00:00:00.000Z"`
`"focusedDate":"2010-02-28T00:00:00.000Z"`